### DISCUSSION TITLE
Thảo luận bài tuần 10 nè

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#13] [General]
Câu hỏi tìm hiểu nè:
  - Bài 7: Máy tính và xã hội tri thức
Câu 1:Có thể thực hiện một phần mềm ứng dụng mà không cần hệ điều hành được không? Vì sao?
Câu 2:Những ứng dụng được cài đặt trên hệ điều hành này có thể cài đặt trên hệ điều hành khác không? Vì sao? Cho ví dụ
Câu 3: Hệ điều hành mở là gì? So sánh giữa ưu, nhược điểm giữa hệ điều hành mở và đóng
  - Bài 8: Ứng dụng của tin học
Câu 1:Hãy kể một số ứng dụng của tin học. Nêu ứng dụng nổi bật yêu thích nhất, giải thích.
Câu 2:Có lĩnh vực nào mà tin học khó có thể sử dụng được?
Câu 3: Tạo một sản phẩm ấn tượng từ sự ứng dụng của tin học.

**Comment by GiaHan06** [comment]
Tui định chia như này nha:
-Bài 7 thì 1 BẠN LÀM CÂU 1 VÀ 2; 1 BẠN LÀM CÂU 3
-Bài 8 thì 1 BẠN LÀM CÂU 1 VÀ 2; 2 BẠN CÙNG LÀM CÂU 3
ai làm câu nào thì rep xuống cái cmt này nha để đỡ bị nhiều tn á

**Reply by GiaHan06** [reply]
tui làm câu 1 2 bài 8 được hong mn??

**Comment by LTGHan** [comment]
Yahooo

**Comment by NguyenNgocQuynhAnh** [comment]
oke tui chốt câu 3 bài 7 nhóe

**Comment by chaudambaotram** [comment]
Vậy tui lam câu 1,2 bài 7 nha

**Comment by HuynhLeBaoTran** [comment]
z tui làm 1, 2 b7 nha

**Comment by HuynhLeBaoTran** [comment]
ủa trùng câu r

**Reply by GiaHan06** [reply]
bà làm câu 3 bài 8 zới Hân nha

**Comment by LTGHan** [comment]
Câu 8c là xao

**Comment by HuynhLeBaoTran** [comment]
câu 3 b8 là jz chòi

**Comment by LTGHan** [comment]
Câu 8c là xaooo

**Comment by LTGHan** [comment]
Àaaaa hiểu rồi

**Comment by LTGHan** [comment]
Nma lm xao để ẤN TƯỢNG?

**Reply by chaudambaotram** [reply]
chắc hợp gu thầy

**Reply by GiaHan06** [reply]
hỏi thầy đi :>
"ũa thầy oi gu thầy là zì zạ??"

**Reply by LTGHan** [reply]
:)))) thầy ơi thầy thích excel hong

**Comment by GiaHan06** [comment]
ulatr nãy h quên tạo project:) sinloi mọi người

**Comment by LTGHan** [comment]
Gòi câu 8c xao....

**Reply by GiaHan06** [reply]
làm như bà nói đi,liệt kê tên stt cả khôi 10 11 12.....

**Reply by LTGHan** [reply]
Ừa đống đó mà ko có bảng tính thì viết tay ha :)))))))))

**Reply by LTGHan** [reply]
Đủ ấn tượng gòi, lên excel nhập stt là ra ha

**Reply by GiaHan06** [reply]
gòi thầy cho 11đ luôn nha

**Comment by HuynhLeBaoTran** [comment]
bất lực...

**Reply by GiaHan06** [reply]
tutu để tui sem

**Reply by GiaHan06** [reply]
alo alo @HuynhLeBaoTran, biết zẽ hong bà

**Reply by GiaHan06** [reply]
zẽ trên máy tính á

**Reply by GiaHan06** [reply]
ũa mà phải ứng dụng tin học hong ta...

**Reply by HuynhLeBaoTran** [reply]
phải á bà

**Comment by LTGHan** [comment]
Oh, tui nhớ là hồi đó có học cái phần mềm j mà nhập lệnh vẽ hình vuông hình tròn í

**Reply by GiaHan06** [reply]
lớp mấy z..

**Reply by LTGHan** [reply]
Nma học hồi tiểu học xa lắc xa lơ quên gòi :')

**Reply by LTGHan** [reply]
> lớp mấy z..

Nhớ lớp 5 j đó

**Comment by HuynhLeBaoTran** [comment]
cho xin í kín nên làm dì đi mn

**Reply by GiaHan06** [reply]
tui đọc trong sách thấy ngta ghi là "ng dùng có thể sử dụng phần mêm máy tính để học zẽ..." mà h hong lẽ vẽ bằng chuột hã?

**Reply by GiaHan06** [reply]
ủa làm poster ha cái gì á có đúng yêu cầu hong

**Reply by HuynhLeBaoTran** [reply]
tui tính làm ds lớp =excel á

**Reply by GiaHan06** [reply]
ulatr thiệt luôn á hã

**Reply by HuynhLeBaoTran** [reply]
đc lun nha

**Reply by HuynhLeBaoTran** [reply]
hic bí quá:((

**Reply by GiaHan06** [reply]
đấy, làm cái chào mừng 20/11 đi

**Reply by GiaHan06** [reply]
@LTGHan alo

**Reply by HuynhLeBaoTran** [reply]
okk đc á

**Reply by HuynhLeBaoTran** [reply]
hợp lun

**Reply by GiaHan06** [reply]
lên canva tham khảo mấy mẫu á, nãy tui tìm cái mẫu 20/11 mà ra đám cưới ròi tùm lum...

**Reply by LTGHan** [reply]
Ngol
